### DISCUSSION TITLE
My profile doesn't calculate pp

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#23396] [Q&A] [answered]
(It may not be quite right to post it here, but it seems to me the quickest and least intrusive way.)

There seems to be a problem with [**my lazer profile**](https://lazer.ppy.sh/users/8543042) (I tried to send an email to peppy a while ago, but maybe it never arrived).
I noticed recently that all my plays on Lazer still don't show the pp obtained, but only the "will be displayed soon" warning sign:

![image](https://user-images.githubusercontent.com/43073074/236219104-28d0e724-3d5b-44e0-9d25-95d860bae9bb.png)

Pinning them seems to show the problem from another angle, claiming that they are worth 0pp:

![image](https://user-images.githubusercontent.com/43073074/236220735-eeb130f0-a549-4009-aa84-1defd551076d.png)

You can see even more of the problem from the stable profile, which shows the `pp next` identical to the stable ones:
![image](https://user-images.githubusercontent.com/43073074/236220025-0358ca22-516a-4b75-b4e5-ec2a5e5ff4f2.png)
(also from the lazer profile:)
![image](https://user-images.githubusercontent.com/43073074/236221001-b52dda92-5e07-4d2f-9ef2-e8d568d60ca8.png)

I checked other threads about it [(1)](https://github.com/ppy/osu/discussions/22352) [(2)](https://github.com/ppy/osu/discussions/23081), and seeing that the problem was being corrected to some people I simply waited in the hope it would happen to me as well, but this doesn't seem to be the case.

I also tried to make some plays on stable a short time ago if it would help in any way, but nothing. I hope you can help me solve the problem.

**Comment by peppy** [comment]
I'm pretty sure this is due to the deflate mod being applied to your plays.

https://github.com/ppy/osu-queue-score-statistics/blob/51fcf88918d60bc3af4a08109df23607c2a11039/osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs#L191-L193

**Reply by Gabixel** [reply]
Oh, I see. I was afraid it was a problem that it was giving this warning, although it still seems to be confusing in my opinion.
So I should be able to get pp in plays without mods (or using the classic ones)?

I hope I didn't misunderstand but, aren't all mods now valid for raking? Or is the mechanism slightly more complex? (Is there a difference between appearing on the leaderboard and getting pp?)
I have found the Hidden with Deflate combination very nice since its release, so I have always used them together.

**Reply by peppy** [reply, marked as answer]
pp is explicitly turned off on all new mods for now, pending further consideration. we did this because some mods make the game easier.
